File: lanto-shqixiu/shwgm/src/main/java/com/lanto/shqixiu/shwgm/model/po/TbCorpDetectInfo.java
package com.lanto.shqixiu.shwgm.model.po;

import java.io.Serializable;

public class TbCorpDetectInfo implements Serializable {
	private Integer stationId;	//STATION_ID
	private String stationName;	//检测站名称
	private String stationNameJc;	//检测站简称
	private String longitude;	//LONGITUDE
	private String latitude;	//LATITUDE
	private String stationTel;	//联系方式
	private String stationFal;	//传真
	private String stationAddress;	//检测站地址
	private String stationArea;	//所属辖区


	public Integer getStationId() {
		return stationId;
	}
	public void setStationId(Integer stationId) {
		this.stationId = stationId;
	}
	public String getStationName() {
		return stationName;
	}
	public void setStationName(String stationName) {
		this.stationName = stationName;
	}
	public String getStationNameJc() {
		return stationNameJc;
	}
	public void setStationNameJc(String stationNameJc) {
		this.stationNameJc = stationNameJc;
	}
	public String getLongitude() {
		return longitude;
	}
	public void setLongitude(String longitude) {
		this.longitude = longitude;
	}
	public String getLatitude() {
		return latitude;
	}
	public void setLatitude(String latitude) {
		this.latitude = latitude;
	}
	public String getStationTel() {
		return stationTel;
	}
	public void setStationTel(String stationTel) {
		this.stationTel = stationTel;
	}
	public String getStationFal() {
		return stationFal;
	}
	public void setStationFal(String stationFal) {
		this.stationFal = stationFal;
	}
	public String getStationAddress() {
		return stationAddress;
	}
	public void setStationAddress(String stationAddress) {
		this.stationAddress = stationAddress;
	}
	public String getStationArea() {
		return stationArea;
	}
	public void setStationArea(String stationArea) {
		this.stationArea = stationArea;
	}

}

File: lanto-shqixiu/shwgm/src/main/webapp/statics/app/model/corp/TbCorpDetectInfo.js
Ext.define('app.model.corp.TbCorpDetectInfo',{
	extend:'jsf.data.VO',
	fields:[
		{name:'STATION_ID' , type:'int', label:'STATION_ID',isShowGrid:false},
		{name:'STATION_NAME' , type:'string', label:'检测站名称',req:true,query: 'lk',isShowGrid:true},
		{name:'STATION_NAME_JC' , type:'string', label:'检测站简称',query: 'lk',isShowGrid:true},
		{name:'LONGITUDE' , type:'string', label:'LONGITUDE',query: 'lk',isShowGrid:true},
		{name:'LATITUDE' , type:'string', label:'LATITUDE',query: 'lk',isShowGrid:true},
		{name:'STATION_TEL' , type:'string', label:'联系方式',isShowGrid:true},
		{name:'STATION_FAL' , type:'string', label:'传真',isShowGrid:true},
		{name:'STATION_ADDRESS' , type:'string', label:'检测站地址',query: 'lk',isShowGrid:true},
		{name:'STATION_AREA' , type:'string', label:'所属辖区',query: 'eq',isShowGrid:true}

		]
});
